apps/explorer/src/pages/validators/Validators.tsx

```diff
@@ -11,7 +11,9 @@ import {
     InfoBox,
     InfoBoxStyle,
     InfoBoxType,
+    Panel,
     TableHeader,
```

Comment:
Lint error
`'TableHeader' is defined but never used. Allowed unused vars must match`